uc-catalog/src/lib.rs

```diff
@@ -63,8 +66,10 @@ impl<'a> UCCatalog<'a> {
             start_version: Some(0),
             end_version: version.and_then(|v| v.try_into().ok()),
         };
-        // TODO: does it paginate?
```

Comment:
Are we just assuming that the client internally handles any pagination? I think yes?